### DISCUSSION TITLE
Deployment Disabled Payment required need help how to do it?

### DISCUSSION THREAD

**Discussion by Vicky396** [vercel/community#2729] [Help]
### Summary

There is some API used in code base by previous developer that now i am working on, its deployment is disabled but i dont know how to renew it. Its using vercel. 
![Screenshot (96)](https://github.com/vercel/community/assets/58840133/8420e721-fddb-4d31-b169-190cc7f0af45)

host: https://www.feedrapp.info/

### Example

https://www.feedrapp.info/

### Steps to Reproduce

Let me know how to pay for API.

**Comment by amyegan** [comment]
Hi @Vicky396. If you own the project, you would need to log into your Vercel account and address any unpaid invoices or other issues with the account. If you are _not_ the project owner, you need to ask the owner to address any account issues or transfer ownership.

**Reply by Vicky396** [reply]
Hi @amyegan I don't have lead on owner of this because the developer made this is shifted to other place so we can't connect with him. Is there any process we can get ownership to this api host?
